apisupport/apisupport.ant/build.xml

```diff
@@ -25,7 +25,7 @@
 
     <target name="nblib" depends="init">
         <mkdir dir="build/antclasses"/>
-        <javac srcdir="antsrc" destdir="build/antclasses" source="1.4" debug="true" deprecation="true">
+        <javac srcdir="antsrc" destdir="build/antclasses" source="1.6" debug="true" deprecation="true">
```

Comment:
The question is whether to update all `source` and `target` to `1.6` unconditionally, or whether to do that only when building on JDK11?

I am fine with unconditional update. Should we need to do it conditionally, then I would define:
`javac.min.source` and `javac.min.target` in `nbbuild/templates/common.xml` and used them as
```xml
source="${javac.min.source}"
```
all over the codebase. Next time the JDK drops support for another version (dropping `1.6` is planned, I think), we would just update `common.xml`. CCing @jlahoda.

Reply:
Actually, JDK 12 already removes support for -source 6, I believe.

I'd suggest to consider using ${javac.source} to use the same source level as the main module sources in cases like this one, where the source code will be run on JDK 8+ anyway.